azure/placeholder-function-app-template.json

```diff
@@ -75,7 +79,7 @@
         },
         {
             "apiVersion": "2021-04-01",
-            "name": "function-app-subnet",
+            "name": "[concat(variables('functionAppName'), '-subnet-', parameters('utcValue'))]",

```

Comment:
we've been using parameters('subnetObject').name for this in other repos. eg https://github.com/SkillsFundingAgency/das-provider-relationships/blob/master/azure/template.json#L157. We should agree on a standard but both options seem sensible